client/grpc-web-client/src/middleware.ts

```diff
@@ -0,0 +1,70 @@
+import {Metadata} from "./metadata";
+import {ProtobufMessage} from "./message";
+import {Code} from "./Code";
+import {MethodDefinition} from "./service";
+import {ClientRpcOptions} from "./client";
+
+const noop = () => void 0;
```

Comment:
heh, for the 3 char saving, I think it's clearer to return `undefined` unless there's something I'm missing?